util/Migrator/DbScripts/2022-12-26_00_OrganizationUserReadByUserIdWithPolicyDetails.sql

```diff
@@ -0,0 +1,46 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE OR ALTER PROCEDURE [dbo].[OrganizationUser_ReadByUserIdWithPolicyDetails]
+    @UserId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+SELECT
+    P.[OrganizationId],
+    P.[Type] as PolicyType,
+    P.[Enabled] as PolicyEnabled,
+    P.[Data] as PolicyData,
+    OU.[Type] as OrganizationUserType,
+    OU.[Status] as OrganizationUserStatus,
+    (CASE WHEN OU.[Permissions] IS NULL THEN 0 ELSE (CASE WHEN COALESCE(JSON_VALUE(OU.[Permissions], '$.managePolicies'), 'false') = 'false' THEN 0 ELSE 1 END) END) as CanManagePolicies,
```

Comment:
Suggestion: we could return the unparsed `Permissions` string to the method. Advantages:
* simplifies SQL
* more flexibility in the `PolicyService` method re: what custom permissions are exempt from what policies
* you could delay the deserialization and only do it if/when you need it (probably not a big deal but thought I'd mention it)

Reply:
Great suggestion, this is now implemented!